CMakeLists.txt

```diff
@@ -111,8 +111,10 @@ cuda_add_library(gdf SHARED
     src/binaryops.cu
     src/bitmaskops.cu
     src/column.cpp
+    src/context.cpp
     src/cudautils.cu
     src/datetimeops.cu
+    src/context.cpp
```

Comment:
The same file should not be added twice, because it will then be compiled and added twice.

Reply:
Fixed.